practices/practice-one/src/index.html

```diff
@@ -31,6 +31,52 @@ <h1 class="heading">Todo list with TypeScript</h1>
                         </div>
                     </div>
                 </section>
+                <ul class="todos">
+                    <!-- add completed class when select button -->
+                    <li class="form-control">
+                        <div class="form-control-icon">
+                            <!--
+                         - add checked class when selected
+                         - also have to add the completed class when adding the checked class
+                        -->
+                            <button class="btn btn-checkbox"></button>
+                        </div>
+                        <!--add edit class when you want to edit-->
+                        <div class="form-control-input">
+                            <p class="pesudo">pham tan duong</p>
+                            <input
+                                class="form-control-input-value"
+                                type="text"
+                                value="pham tan duong"
```

Comment:
Why hard code for this?

Reply:
It's just sample data, I'll delete it later